content/doc/book/system-administration/administering-jenkins-on-kubernetes.adoc

```diff
@@ -12,4 +12,87 @@ ifdef::backend-html5[]
+
+* Fix what needs to be fixed, and deploy the changes.
+
+You may now start the upgrade of Kubernetes.
+The process has two steps: 
+. Upgrade the control plane.
+. Upgrade the nodes. 
+
+Follow the upgrade steps according to your cloud provider.
+* link:https://docs.aws.amazon.com/eks/latest/userguide/update-cluster.html[Amazon]
+* link:https://learn.microsoft.com/en-us/azure/aks/upgrade-cluster?tabs=azure-cli[Azure]
+* link:https://cloud.google.com/kubernetes-engine/docs/how-to/upgrading-a-cluster[Google]
+* Pour yourself a cup of your favorite drink while you sit back and wait for the magic to happen.
+
+== Kubernetes Concepts:
+
+The Concepts section helps you learn about the parts of the Kubernetes system, the abstractions Kubernetes uses to represent your cluster, and helps you obtain a deeper understanding of how Kubernetes works.
+
+link:https://kubernetes.io/docs/concepts/overview/[Overview]: Kubernetes is a portable, extensible, open source platform for managing containerized workloads and services, that facilitates both declarative configuration and automation. 
+It has a large, rapidly growing ecosystem.
+Kubernetes services, support, and tools are widely available.
+
+* link:https://kubernetes.io/docs/concepts/architecture/[Cluster Architecture]: The architectural concepts behind Kubernetes.
+
+* link:https://kubernetes.io/docs/concepts/containers/[Containers]: Technology for packaging an application along with its runtime dependencies.
+
+* link:https://kubernetes.io/docs/concepts/windows/[Windows in Kubernetes]
+
+* link:https://kubernetes.io/docs/concepts/workloads/[Workloads]: Understand Pods, the smallest deployable compute object in Kubernetes, and the higher-level abstractions that help you to run them.
+
+* link:https://kubernetes.io/docs/concepts/services-networking/[Services, Load Balancing, and Networking]: Concepts and resources behind networking in Kubernetes.
+
+* link:https://kubernetes.io/docs/concepts/storage/[Storage]: Ways to provide both long-term and temporary storage to Pods in your cluster.
+
+* link:https://kubernetes.io/docs/concepts/configuration/[Configuration]: Resources that Kubernetes provides for configuring Pods.
+
+* link:https://kubernetes.io/docs/concepts/security/[Security]: Concepts for keeping your cloud-native workload secure.
+
+* link:https://kubernetes.io/docs/concepts/policy/[Policies]: Policies you can configure that apply to groups of resources.
+
+* link:https://kubernetes.io/docs/concepts/scheduling-eviction/[Scheduling, Preemption and Eviction]: In Kubernetes, scheduling refers to making sure that Pods are matched to Nodes so that the kubelet can run them.
+Preemption is the process of terminating Pods with lower priority so that Pods with higher priority can schedule on Nodes.
+Eviction is the process of proactively terminating one or more Pods on resource-starved Nodes.
```

Comment:
```suggestion
* link:https://kubernetes.io/docs/concepts/scheduling-eviction/[Scheduling, Preemption and Eviction]: In Kubernetes, scheduling refers to making sure that Pods are matched to Nodes so that the kubelet can run them.
Preemption is the process of terminating Pods with lower priority so that Pods with higher priority can schedule on Nodes.
Eviction is the process of proactively terminating one or more Pods on resource-starved Nodes.
```
For this bullet, would there be a way to condense the description? The other items in this list have been summarized very well, so maybe something like:

Pod deployment and maintenance concepts in Kubernetes.

This is only a potential replacement, if there is another phrasing you'd like to use, please do so!